client/app/hosting/ftp/FTP.html

```diff
@@ -1,343 +1,296 @@
 <div class="container-fluid px-0" data-ng-controller="HostingTabFTPCtrl as ctrlHostingTabFTP">
 
-    <div class="text-center" data-ng-if="ctrlHostingTabFTP.loading.init">
-        <oui-loader size="l" inline="true"></oui-loader>
-    </div>
-
-    <div class="row" data-ng-show="!ctrlHostingTabFTP.loading.init">
-
-        <!-- Doesnt have multiftp option -->
-        <div class="col-md-9">
-            <div data-ovh-alert="{{alerts.main}}"></div>
-
-            <p data-i18n-static="hosting_tab_FTP_dashboard_information"></p>
-
-            <!-- PERSO  -->
-            <div data-ng-if="ctrlHostingTabFTP.ftpInformations.hasMultiFtp === false">
-                <dl>
-                    <dt data-i18n-static="hosting_tab_FTP_dashboard_server"></dt>
-                    <dd class="mb-3">
-                        <a data-ng-href="ftp://{{ctrlHostingTabFTP.firstUser.ftp.user}}@{{ctrlHostingTabFTP.firstUser.ftp.url}}:{{ctrlHostingTabFTP.firstUser.ftp.port}}/" target="_blank" title="{{i18n.hosting_dashboard_service_home_ftp_newtab}}">
-                            <span data-ng-bind="ctrlHostingTabFTP.firstUser.ftpUrl"></span>
-                            <span class="fa fa-external-link" aria-hidden="true"></span>
-                        </a>
-                    </dd>
+        <div class="loader mt-5 text-center" aria-label="{{i18n.common_loading}}" data-ng-if="ctrlHostingTabFTP.loading.init"></div>
```

Comment:
`<oui-loader size="l" data-ng-if="ctrlHostingTabFTP.loading.init"></oui-loader>` ?

Careful on the indentation of the file. The diff is "broken".

Reply:
The indentation will be fixed in a future commit :)